src/load-config.ts

```diff
@@ -5,10 +5,12 @@ import {Interfaces, Config} from '@oclif/core'
  * @param {loadConfig.Options} opts options
  * @return {Promise<Interfaces.Config>} config
  */
-export function loadConfig(opts: loadConfig.Options = {}) {
+export function loadConfig(opts: loadConfig.Options = {}): { run(ctx: { config: Interfaces.Config}): Promise<Interfaces.Config> } {
   return {
     async run(ctx: {config: Interfaces.Config}) {
       ctx.config = await Config.load(opts.root || loadConfig.root)
+      // Always set the topicSeparator to a space so that users can write their tests using either spaces or colons
+      ctx.config.topicSeparator = ' '
```

Comment:
Would this cause tests written with spaces to succeed even if the config for the CLI/plugin uses `:` as a topic separator?

Reply:
@RodEsp Yes, that's correct. I figured that this would be a more user friendly approach than asking users to specify the topic separator in every single test

Reply:
That would be producing false positives for tests though because actually running the command with spaces in that situation would fail.

Does the user have to specify the separator per test? Having it listed in their root level `package.json` isn't enough?